larq_compute_engine/mlir/tests/prepare-tf.mlir

```diff
@@ -12,7 +12,7 @@ func @bsign(%arg0: tensor<8x16xf32>) -> tensor<8x16xf32> {
 }
 
 func @bconv2d(%arg0: tensor<1x112x112x2xf32>) -> tensor<1x112x112x2xf32> {
-  %cst = constant dense<[[[[1.0, -1.0], [1.0, 1.0]], [[-1.0, 1.0], [-1.0, 1.0]]]]> : tensor<1x2x2x2xf32>
+  %cst = "tf.Const"() { value = dense<[[[[1.0, -1.0], [1.0, 1.0]], [[-1.0, 1.0], [-1.0, 1.0]]]]> : tensor<1x2x2x2xf32> } : () -> tensor<1x2x2x2xf32>
```

Comment:
This change is not required to make the tests pass, but I think it is a good idea to test that a `tf.Const` correctly gets matched by the transform as well since the transform no matches a MLIR `ConstantOp` instead of the TFLite and TF versions.